### PR TITLE
WIP: Demo website - Django backend

### PR DESCRIPTION
**Pull request by darthgera123**
Basic Page made.Along with that i made the backend tables and demonstrated how they can look. I defined the relationships b/w events and other tables. Followed the doc and djangogirls website for inspiration. Work in progress. Also solves #9

### CODE REVIEW

**Comment by vishwarajanand** [comment]
Two suggestions:

1. Add screenshots to a `screenshots/` folder so that we can better review your changes and correlate with code.

2. Move all your code and screenshots  to `poc/website/darthgera123`

Once finished making changes, please remove `[WIP]` in the description so that we know when to start the review.

**Comment by darthgera123** [comment]
will do

File: girlscript/girlscript/settings.py [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/2.0/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = '%&@imstkn8a#w$91i#0uitz83rkft%l*m1dfik9246^)12l_95'
```
Comment (0x48piraj) [inline girlscript/girlscript/settings.py]: Why leaking the key?
Why not using env vars

**Review by 0x48piraj — COMMENTED** [review]
Seems good.

**Comment by 0x48piraj** [comment]
Also, please squash the commits.

File: girlscript/girlscript/settings.py (hunk 1 above)
Comment (darthgera123) [reply girlscript/girlscript/settings.py]: i was making a demo website so didnt really look into that

Comment (0x48piraj) [reply girlscript/girlscript/settings.py]: That's a valid point. Just make sure to fix this if your website gets accepted. :)

**Comment by vishwarajanand** [comment]
Any update?

**Comment by darthgera123** [comment]
i am working on the list i made. backend will be mostly done by tomorrow

**Comment by vishwarajanand** [comment]
@darthgera123 You can skip much of the UI/CSS stuff. We just want a demo for how the functionality of adding chapters would work. We would anyway need to change the CSS for the final website.

We really want to move to the integration of POC soon but are stuck on seeing this merged. Could you please expedite it or let us know where you are stuck?

**Comment by darthgera123** [comment]
ill complete by tonight, sorry was busy. could you elaborate on integration of POC? what all feature do you want with a certain chapter(as such what all features, wasnt mentioned in the blog)

**Comment by vishwarajanand** [comment]
Could you elaborate on the integration of POC? 
> You can ignore the integration for now. We would do it separately. 

What all feature do you want with a certain chapter?
> For now a simple page with chapter name would be fine.

**Comment by vishwarajanand** [comment]
@darthgera123 If you are unable to do the further development, let us know. We would reassign this to someone else or close this PR so that at least the work doesn't get stuck.

**Comment by darthgera123** [comment]
I'm so sorry for being non responsive. I will complete it by wednesday

**Comment by vishwarajanand** [comment]
Since we are approaching the project's deadline, we would have to close this PR by tomorrow, if not completed.

**Pull request closed without merging**